dataloader/dataloader.go

```diff
@@ -0,0 +1,55 @@
+package dataloader
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"../defination"
+	"../interfaces"
+)
+
+const (
+	getPostsEndpoint    = "https://my-json-server.typicode.com/typicode/demo/posts"
+	getCommentsEndpoint = "https://my-json-server.typicode.com/typicode/demo/comments"
+)
+
+type SeviceLoader struct {
+	data interfaces.LoadData
+}
+
+func (service *SeviceLoader) GetPosts() ([]defination.Post, error) {
```

Comment:
Nitpick: this function does 2 things (get data and rendering), so can be separated into 2 smaller functions for readability